style.css

```diff
@@ -415,6 +427,51 @@ nav {
   outline-color: #396df2;
 }
 
+.popup {
```

Comment:
- [ ] KIndly check the requirements on the image
Yours:
![image](https://user-images.githubusercontent.com/62705947/209170557-fb8aaea4-d30c-47e2-8abf-2a43dd0e5e07.png)
Design:
![image](https://user-images.githubusercontent.com/62705947/209170662-1fb2db56-b6d5-4f13-94c4-432bc3b7d4c1.png)
- PS: Same requirements go for mobile version as well